utils/releaser/src/ReleaseWorker/Release/CreateAndPushGitTagsExceptProjectBaseReleaseWorker.php

```diff
@@ -0,0 +1,114 @@
+{
+    /**
+     * Packages that are not released - old packages or forks
+     * @var string[]
+     */
+    private const EXCLUDED_PACKAGES = [
+        // excluded from the initial tagging as there needs to be another commit with composer.lock and package-lock.json
+        // @see https://github.com/shopsys/shopsys/pull/1264
+        'shopsys/shopsys',
+        'shopsys/project-base',
+        // not maintained anymore
+        'shopsys/product-feed-interface',
+        'shopsys/phpstorm-inspect',
+        // forks
+        'shopsys/postgres-search-bundle',
+        'shopsys/doctrine-orm',
+        'shopsys/jparser',
+        // old packages
+        'shopsys/syscart',
+        'shopsys/sysconfig',
+        'shopsys/sysreports',
+        'shopsys/sysstdlib',
+    ];
+
+    /**
+     * @var \Shopsys\Releaser\Packagist\PackageProvider
+     */
+    private $packageProvider;
+
+    /**
+     * @param \Shopsys\Releaser\Packagist\PackageProvider $packageProvider
+     */
+    public function __construct(PackageProvider $packageProvider)
+    {
+        $this->packageProvider = $packageProvider;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Create and push git tags for packages excluding monorepo and project-base';
+    }
+
+    /**
+     * Higher first
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 640;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        $packages = $this->packageProvider->getPackagesByOrganization('shopsys', self::EXCLUDED_PACKAGES);
+        $packages = str_replace('shopsys/', '', $packages);
+
+        $versionString = $version->getVersionString();
+
+        $this->processRunner->run('mkdir temp');
+        $packageNamesWithProblems = [];
+        foreach ($packages as $packageName) {
+            $this->processRunner->run('cd temp && git clone https://github.com/shopsys/' . $packageName . '.git');
+            $this->processRunner->run('cd temp/' . $packageName . ' && git checkout master && git tag ' . $versionString);
+            $this->processRunner->run('cd temp/' . $packageName . ' && git log --graph --oneline --decorate=short --color | head', true);
+            $pushTag = $this->symfonyStyle->ask('Is the tag on right commit and should be pushed?', 'yes');
+
+            if ($pushTag !== 'yes') {
+                $packageNamesWithProblems[] = $packageName;
+            }
+        }
+
+        if (count($packageNamesWithProblems) === 0) {
+            foreach ($packages as $packageName) {
+                $this->processRunner->run('cd temp/' . $packageName . ' && git push origin ' . $versionString);
+            }
+
+            $this->processRunner->run('rm -r temp');
```

Comment:
I didn't remove the `temp` directory (could happen on some error), and the following worker `CreateAndCommitComposerLockAndPackageLockReleaseWorker` commited them. I really recommend different temp directory